docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md

```diff
@@ -0,0 +1,128 @@
+<!--
+title: "Troubleshooting Agent with Cloud connection"
+description: "A simple guide to troubleshoot occurances where the Agent is showing as offline after claiming."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md
+-->
+
+# Troubleshooting Agent with Cloud connection
+
+Sometimes, when claiming a node, it might not show up as online in Netdata Cloud.  
+The occurances triggering this behavior might be:
+
+- [The claiming script failed](#the-claiming-script-failed)
+- [Claiming on an older, deprecated version of the Agent](#claiming-on-an-older-deprecated-version-of-the-agent)
+- [Network issues while connecting to the Cloud](#network-issues-while-connecting-to-the-cloud)
+
+## The claiming script failed
+
+### Make sure the Agent is running
+
+Check if the Agent is running:
+
+```bash
+systemctl status netdata
+```
+
+The expected output should contain info like this:
+
+```bash
+Active: active (running) since Wed 2022-07-06 12:25:02 EEST; 1h 40min ago
+```
+
+:::note
+
+The Agent must be running for the claiming to work.
+
+:::
+
+If the Agent is not running, enable the service:
+
+```bash
+sudo systemctl enable netdata 
+```
+
+and then start the Agent:
+
+```bash
+sudo systemctl start netdata 
+```
+
+:::info
+
+Read more about [Starting, Stopping and Restarting the Agent](https://learn.netdata.cloud/docs/configure/start-stop-restart).
+
+:::
+
+## Claiming on an older, deprecated version of the Agent
+
+Make sure that you are using the latest version of Netdata if you are using the [Claiming script](https://learn.netdata.cloud/docs/agent/claim#claiming-script).
+
+With the introduction of our new architecture, Agents running versions lower than `v1.32.0` can face claiming problems, so we reccomend to [update the Netdata Agent](https://learn.netdata.cloud/docs/agent/packaging/installer/update).
```

Comment:
reccomend -> recommend

Reply:
maybe also note that the recommendation is to be on the latest netdata version, in any case they should be on latest stable.